### PR TITLE
Sign of life - test module

### PR DESCRIPTION
**Pull request by Pokisiekk**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Dasharo compatibility: Sign of life
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+## SOL001.001 Detection of Dasharo firmware and Dasharo EC firmware
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md:10]: @Pokisiekk I think we should select appropriate names for these two test cases. 

At this point, as if I had glanced at the name alone, I would have suspected that the test requires the execution of some commands that will return output containing information about the firmware and EC versions.

So I would suggest:
1. SOL001.001 SOL string shows Dasharo firmware and EC version
2. SOL002.001 SOL string shows information about proprietary EC

File: docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Note the results.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md:31]: To make the test correspond to the planned automatic tests, we should add the information:

```
2. Wait for Sign of life string
```

File: docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md:66]: As above

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md]: Updated

File: docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md:68]: Added

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**